plugins/woo-ai/src/shared/random-loading-message.tsx

```diff
@@ -0,0 +1,216 @@
+/**
```

Comment:
Lets just make sure we run all this phrasing by Jarek, I like the whimsiness but we want to make sure it matches the aesthetic that Woo is going for.